WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -151,4 +153,51 @@ public void onCancel(DialogInterface dialog) {
         // Do not ask again
         AppPrefs.setImageOptimizePromoRequired(false);
     }
+
+    /**
+     * Given a media error returns the error message to display on the UI.
+     *
+     * @param error The media error occurred
+     * @return String  The associated error message.
+     */
+    public static String getErrorMessage(final Context context, boolean suggestMediaOptimization, final MediaModel media, final MediaStore.MediaError error) {
```

Comment:
The parameter `suggestMediaOptimization` seems like a misnomer (and I not sure the logic below is correct) since what's being passed is whether image optimization is enabled for the site. Can we rename this `isImageOptimizationEnabled`?